docs/tools/glaze/glossary.md

```diff
@@ -0,0 +1,29 @@
+# Glaze Glossary
+
+Common terms used in Glaze packages and tools.
+
+## Schema
+
+A Ceramic TileDocument storing a JSON schema as contents. Schemas are the basis of most Glaze packages and tools functionalities.
+
+## Definition
+
+A TileDocument matching the Definition spec from [CIP-11](https://github.com/ceramicnetwork/CIP/blob/main/CIPs/CIP-11/CIP-11.md). Definitions are used in the [Index](#index) of the DID DataStore.
+
+## Tile
+
+In the context of Glaze packages, a Tile refers to a TileDocument with an associated [Schema](#schema).
+
```

Comment:
These seem like duplicates from the general Ceramic glossary. Maybe we can combine the two?

Reply:
Yes we could merge the two if we want, I think it's easier to have a more specific scope to the Glaze packages but I don't mind merging as a section of the main glossary if it's better.

Reply:
@michaelsena input here would be good